1_fetch.yml

```diff
@@ -44,9 +45,16 @@ targets:
     command: viz_config[[I('dates')]]
   stream_params:
     command: yaml.load_file('1_fetch/cfg/stream_params.yml')
+  precip_config:
+    command: viz_config[I(c("precip_cols", "precip_bin"))]
```

Comment:
' " ?!? 😠

Reply:
killin' me here

Reply:
It's so refreshing to work with people even more OCD than i am =)